src/AVFoundation/AVAssetDownloadStorageManagementPolicy.cs

```diff
@@ -0,0 +1,30 @@
+#if IOS
+
+using System;
+
+using XamCore.Foundation;
+using XamCore.ObjCRuntime;
+
+namespace XamCore.AVFoundation {
+	public partial class AVAssetDownloadStorageManagementPolicy {
+
+		[iOS (11,0)]
+		[NoTV][NoMac][NoWatch]
```

Comment:
I have not finished my email backlog and this might be answered there but shouldn't we be using the full availability attributes instead?

Reply:
No, too long to type (or even read). The attributes now exists (not macros) since `pmcs` was removed. Also one (of the) thing(s) that we (@chamons first) learned is that using different ways to express the same thing is cause for future pain.

Reply:
Ahh this is great news that we now have those attributes ❤️ it!